### DISCUSSION TITLE
Finding saved chats

### DISCUSSION THREAD

**Discussion by tomco582** [brianpetro/obsidian-smart-connections#314] [General]
I see in Smart Connections after a chat that there is a "save chat" button. My question is: once the chat is saved how can it be retrieved. Where are files.

**Comment by RealRavens** [comment]
[![Image from Gyazo](https://i.gyazo.com/5def72816ec16055f9c0b38eb991f811.png)](https://gyazo.com/5def72816ec16055f9c0b38eb991f811)

So on the left (where it says UNTITLED) is a space for you to give the chat file a name (to help you get back to it). To the right of the save (floppy disk) icon is Chat history button. You can click that and it will bring up a list of your previously saved chats. Click on it and you will proceed where you left off in that chat.

The location of chats in your system explorer (assuming windows) is in the root of your vault if you want to access the files directly.

So for example you may see something like: Obsidian Vault/Your-Vault-Name/.smart-connections/chats...

There will be a collection of json files which are all the chats that you've been having.
